students/10th/kimjungho/style/login.css

```
@@ -0,0 +1,73 @@
+
+input {
+    margin: 0 auto;
+    border: 1px solid lightgray;
+    border-top-left-radius: 10px; /* 왼쪽 위 */
+    border-top-right-radius: 10px; /* 오른쪽 위 */
+    border-bottom-right-radius: 10px; /* 오른쪽 아래 */
+    border-bottom-left-radius: 10px; /* 왼쪽 아래 */
+  }
+  
+  
+  
+  
+  .login-container {
+      /*padding: 50px 10px 50px 10px;*/
+      margin-top: 300px;
+      margin-right: auto;
+      margin-left: auto;
+      margin-bottom: auto;
+      display: flex;
+      max-height: 800px;
+      max-width: 500px;
+      /*height: 50vh;*/
+      border: 2px solid lightgray;
+      flex-direction: column;
+      justify-content: space-between;
+      align-items: center;
+  }
+  
+  
+  .login-form {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+  }
+  
+  div {
+      
+      padding: 2px;
+      border-radius: 10px;
+      border-color: lightgray;
+      text-align: center;
+  }
+  
+  input{
+      height: 30px;
+  }
+  
+  
+  .forgetpw {
+          color: #00376b;
+          color: rgba(var(--fe0,0,55,107),1);
+          font-size: 12px;
+          line-height: 14px;
+          margin-top: 50px;
+          margin-right: auto;
+          margin-left: auto;
+          text-align: center;
+      }

```

```suggestion
input {
  height: 30px;
}
  
.forgetpw {
  color: #00376b;
  font-size: 12px;
  line-height: 14px;
  margin: 50px auto 0;
  text-align: center;
})
```

1. 스타일과 스타일 사이는 한칸만 띄워주세요 (input, .forgetpw)
2. color가 중복되어 있네요. 이 경우 뒤에 나온 속성이 우선 적용됩니다. 알맞은 색상을 골라 하나만 적용해주세요.
3. margin에 단축 속성을 적용할 수 있겠습니다.
4. indent 일정하게 지켜주세요!

---

넵 수정하여 반영하겠습니다 👍